_includes/index.html

```diff
@@ -493,6 +494,66 @@ <h1><a name="data">{{ 'data.title' | translate }}</a></h1>
     </p>
   </article>
 
+  <article>
+    <h1><a name="separate">{{ 'separate.title' | translate }}</a></h1>
+
+    <p>{{ 'separate.body.first' | translate }}</p>
+    <div class='code-block'>
+      <p class="spec-title spec-wrong">{{ 'shared.bad_code' | translate }}</p>
+
+{% highlight ruby %}
+describe 'User' do
+  describe '#fiddle' do
+    it 'lets the user play their fiddle' do
+      user = User.create
+      fiddle = instance_double Fiddle
+      allow(Fiddle).to receive(:new).and_return(fiddle)
+      expect(user.fiddle).to eq("🎵")
+    end
+  end
+end
+{% endhighlight %}
+    </div>
+    <div class='code-block'>
+      <p class="spec-title spec-correct">{{ 'shared.good_code' | translate }}</p>
+
+{% highlight ruby %}
+describe 'User' do
+  subject(:user) { User.create }
+
+  describe '#fiddle' do
+    let(:fiddle) { instance_double Fiddle }
+    before { allow(Fiddle).to receive(:new).and_return(fiddle) }
+
+    it 'lets the user play their fiddle' do
+      expect(user.fiddle).to eq("🎵")
+    end
+  end
+end
+{% endhighlight %}
+    <p>{{ 'separate.body.second' | translate }}</p>
+    <div class="code-block">
+      <p class="spec-title spec-correct">{{ 'shared.good_code' | translate }}</p>
+
+{% highlight ruby %}
+let(:in_tune) { true }
+
+it { is_expected.to eq("🎵")
+
+context 'when the fiddle needs a tune up' do # this describes a variation
+  let(:in_tune { false } # this creates the variation
+
+  it { is_expected.to eq("😬")
+end
+{% endhighlight %}
+    </div>
+    <p>
+      <a href="https://github.com/andreareginato/betterspecs/issues/11">
```

Comment:
I'll change this link once an appropriate issue is created. If I need to proactively do something like that please let me know!